backend/config.py

```
@@ -37,7 +37,7 @@ class Settings(BaseSettings):
     rate_limit_period: int = 3600
 
     # Security
-    secret_key: str = "your-secret-key-here-change-in-production"
+    secret_key: str = "dev-secret-key-change-in-production"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "secret_key"

(S105)